package/MDAnalysis/topology/ITPParser.py

```diff
@@ -268,7 +268,7 @@ def find_path(self, path):
             path = path.name
```

Comment:
A slightly different change seems to allow the testsuite to pass while still allowing relative file path handling in my hands:

```diff
--- a/package/MDAnalysis/topology/ITPParser.py
+++ b/package/MDAnalysis/topology/ITPParser.py
@@ -265,7 +265,7 @@ class GmxTopIterator:
             current_file = self.current_file
 
         try:
-            path = path.name
+            path = os.path.abspath(path.name)
         except AttributeError:
             pass
```

I think it addresses Irfan's comment as well